### PR TITLE
feat/post update

### PR DESCRIPTION
**Pull request by YoshkiMatryoshki**
As a team we proud to announce that with the new updates we bring to you UNIQUE and REFRESHING post update logic into server which is 50% more **BLAZINGLY FASTER** than logic from previous iterations! TRULY REMARKABLE, preorder patch requests**_(tm)_** with "Approve" button just for 1999.99$

### CODE REVIEW

**Comment by tikitko** [comment]
> As a team we proud to announce that with the new updates we bring to you UNIQUE and REFRESHING post update logic into server which is 50% more faster than logic from previous iterations! TRULY REMARKABLE, preorder patch requests**_(tm)_** with "Approve" button just for 1999.99$

"blazingly fast"

File: blog-server-api/src/endpoints/create_post/response_content_failure.rs [hunk 1]
```diff
@@ -30,7 +30,7 @@ impl ApiResponseContentFailure for CreatePostContentFailure {
             CreatePostContentFailure::ValidationError { reason: _ } => {
                 "CREATE_POST_VALIDATION_ERROR"
             }
-            CreatePostContentFailure::AlreadyExists => "CREATE_POST_ALREASY_EXISTS",
```
Comment (tikitko) [inline blog-server-api/src/endpoints/create_post/response_content_failure.rs:33]: 🤡

**Review by tikitko — CHANGES_REQUESTED** [review]

**Comment by tikitko** [comment]
you

**Comment by YoshkiMatryoshki** [comment]
> > As a team we proud to announce that with the new updates we bring to you UNIQUE and REFRESHING post update logic into server which is 50% more faster than logic from previous iterations! TRULY REMARKABLE, preorder patch requests**_(tm)_** with "Approve" button just for 1999.99$
> 
> "blazingly fast"

Absolutely! Description changed

File: blog-server-api/src/endpoints/create_post/response_content_failure.rs (hunk 1 above)
Comment (YoshkiMatryoshki) [reply blog-server-api/src/endpoints/create_post/response_content_failure.rs:33]: yep

File: blog-server-api/src/endpoints/update_post/response_content_success.rs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    fn status_code(&self) -> &'static StatusCode {
+        &StatusCode::OK
+    }
+}
+
+impl ApiResponseContentSuccess for UpdatePostContentSuccess {
+    type Data = Self;
+
+    fn identifier(&self) -> &'static str {
+        "POST_UPDATED"
```
[the remark this answers is not in the document]
Comment (YoshkiMatryoshki) [reply blog-server-api/src/endpoints/update_post/response_content_success.rs]: done

**Review by tikitko — APPROVED** [review]

**Pull request merged**